packages/govuk-frontend-review/src/app.mjs

```diff
@@ -48,8 +48,7 @@ export default async () => {
 
   // Feature flags
   const flags = /** @type {FeatureFlags} */ ({
-    isDeployedToHeroku: !!process.env.HEROKU_APP,
-    isDevelopment: !['test', 'production'].includes(process.env.NODE_ENV),
+    isDevelopment: !process.env.HEROKU_APP && !process.env.CI,
```

Comment:
Shall we make this so either can be set independently?

Reply:
I don't quite follow – do you mean have separate flags for Heroku and for CI?

Reply:
Sorry, no, but that really wasn't very well explained from me

For example, to start the Review app "from cache" locally you can do:

```console
HEROKU_APP=review npm start --workspace @govuk-frontend/review
```

But previously you could use `NODE_ENV=development` to flip to `isDevelopment: true`

```console
NODE_ENV=development HEROKU_APP=review npm start --workspace @govuk-frontend/review
```

Think it matters?

```suggestion
    isDevelopment:
      process.env.NODE_ENV === 'development' ||
      (process.env.CI !== 'true' && !process.env.HEROKU_APP),
```

Reply:
At the minute we'll always default `isDevelopment` to true unless either `HEROKU_APP` or `CI` are set.

What are the scenarios where those environment variables might be set but we need to override them? I can't think of anything obvious off the top of my head…

Reply:
Only what we briefly talked about when pairing

E.g. To run a Heroku preview in "development mode" for extra debugging or log output etc

You've answered the "Shall we…?" and it's not a blocker so can leave it